docs/index.rst

```diff
@@ -243,6 +242,20 @@ Indicate a specific time granularity (or multiple):
     >>> future.humanize(present, only_distance=True, granularity=["hour", "minute"])
     'an hour and 6 minutes'
 
+Indicate shortened time:
+
+.. code-block:: python
+
+    >>> present = arrow.utcnow()
+    >>> future = present.shift(hours=2)
+    >>> future.humanize(present)
+    'in 2 hours'
+    >>> future.humanize(present, brief=True)
+    '2h'
+    >>> future = present.shift(minutes=66)
+    >>> present.humanize(future, granularity=["hour", "minute"], brief=True)
+    'h 6m'
```

Comment:
Can't say I'm a fan of dropping the `1` here.

Reply:
^ Ditto